esp/esdllib/esdl_script.cpp

```diff
@@ -98,9 +101,11 @@ class CEsdlTransformOperationVariable : public CEsdlTransformOperationBase
 public:
     CEsdlTransformOperationVariable(IPropertyTree *tree, const StringBuffer &prefix) : CEsdlTransformOperationBase(tree, prefix)
     {
+        if (m_traceName.isEmpty())
+            m_traceName.set(tree->queryProp("@name"));
```

Comment:
This seems to be redundant, to the point that it could probably make sense moving in to the base class. If the base class handles m_traceName, it makes sense for it to handle fall-back options (could maybe add an optional fall-back name to the constructor parameter list to not assume).

I'm not sure I would go so far as to move storage for @name into the base, although I probably wouldn't complain if it happened.

Reply:
I'll take a look but @name is only a fallback for trace on commands where it is already expected.  name is too valuable as a parameter to use it to mean trace.  So I want to avoid that in the future.  When dealing with name + value situations it's a shame if we can't make it intuitive because of how we did tracing.